content/app/guides/payment/backend-configuration/_index.nb.md

```diff
@@ -0,0 +1,170 @@
+
+```xml
+    <bpmn:startEvent id="StartEvent_1">
+      <bpmn:outgoing>Flow_start_t1</bpmn:outgoing>
+    </bpmn:startEvent>
+
+    <bpmn:sequenceFlow id="Flow_start_t1" sourceRef="StartEvent_1" targetRef="Task_1" />
+
+    <bpmn:task id="Task_1" name="Utfylling">
+      <bpmn:incoming>Flow_start_t1</bpmn:incoming>
+      <bpmn:incoming>Flow_g1_t1</bpmn:incoming>
+      <bpmn:outgoing>Flow_t1_t2</bpmn:outgoing>
+      <bpmn:extensionElements>
+        <altinn:taskExtension>
+          <altinn:taskType>data</altinn:taskType>
+        </altinn:taskExtension>
+      </bpmn:extensionElements>
+    </bpmn:task>
+
+    <bpmn:sequenceFlow id="Flow_t1_t2" sourceRef="Task_1" targetRef="Task_2" />
+
+    <bpmn:task id="Task_2" name="Betaling">
+      <bpmn:incoming>Flow_t1_t2</bpmn:incoming>
+      <bpmn:outgoing>Flow_t2_g1</bpmn:outgoing>
+      <bpmn:extensionElements>
+        <altinn:taskExtension>
+          <altinn:taskType>payment</altinn:taskType>
+          <altinn:actions>
+            <altinn:action>confirm</altinn:action>
+            <altinn:action>pay</altinn:action>
+            <altinn:action>reject</altinn:action>
+          </altinn:actions>
+          <altinn:paymentConfig>
+            <altinn:paymentDataType>paymentInformation</altinn:paymentDataType>
+          </altinn:paymentConfig>
+        </altinn:taskExtension>
+      </bpmn:extensionElements>
+    </bpmn:task>
+
+    <bpmn:sequenceFlow id="Flow_t2_g1" sourceRef="Task_2" targetRef="Gateway_1" />
+
+    <bpmn:exclusiveGateway id="Gateway_1">
+      <bpmn:incoming>Flow_t2_g1</bpmn:incoming>
+      <bpmn:outgoing>Flow_g1_t1</bpmn:outgoing>
+      <bpmn:outgoing>Flow_g1_end</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+
+    <bpmn:sequenceFlow id="Flow_g1_t1" sourceRef="Gateway_1" targetRef="Task_1">
+      <bpmn:conditionExpression>["equals", ["gatewayAction"], "reject"]</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_g1_end" sourceRef="Gateway_1" targetRef="EndEvent_1">
+      <bpmn:conditionExpression>["equals", ["gatewayAction"], "confirm"]</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+
+    <bpmn:endEvent id="EndEvent_1">
+      <bpmn:incoming>Flow_g1_end</bpmn:incoming>
+    </bpmn:endEvent>
+```
+NB: Verdien til denne noden: `<altinn:paymentDataType>paymentInformation</altinn:paymentDataType>` må matche ID-en til datatypen du konfigurerte i forrige steg.
+
+### 3. Gi tilganger til den som skal betale:
+
+Brukeren som skal betale må ha rettigheter til `read`, `write`, `pay`, `confirm` og `reject` handlingene på betalingprosessteget.
+
+
+### 4. Implementer IOrderDetailsCalculator interfacet i C#:
+
+Legg til en ny klasse der du har din custom kode, f.eks:  `App/logic/OrderDetailsCalculator.cs`.
+
+Her vil du implementere din logikk for å kalkulere hva brukeren skal betale for.
+Du kan for eksempel aksessere skjemadata, legge til obligatoriske avgifter, eller kun legge til en fast kostnad for skjemaet. 
```

Comment:
Du kan for eksempel få tilgang til skjemadata, legge til obligatoriske avgifter, eller kun legge til en fast kostnad for skjemaet.